r/R/enums.R

```diff
@@ -79,6 +79,8 @@ Type <- enum("Type::type",
   LARGE_STRING = 34L,
   LARGE_BINARY = 35L,
   LARGE_LIST = 36L
+  INTERVAL_MONTH_DAY_NANO = 37L,
```

Comment:
Thanks for fixing this! Just missing a comma on the line above this one, and then hopefully tests should pass.

Reply:
Thank you! I fixed it and pushed again.